<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonCreateMenu</class>
 <widget class="QWidget" name="PersonCreateMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_EnterName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_EnterBirthYear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter year of birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_GenderPicker">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>85</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Gender:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_Name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l_EnterDeathYear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter year of death (if dead):</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Contribution">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;   Contribution of the &lt;br/&gt;	person:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="input_Contribution">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_TuringYear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Turing Year:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_TuringYear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>145</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1966</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1967</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1968</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1969</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1970</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1971</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1972</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1973</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1974</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1975</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1976</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1977</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1979</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1980</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1981</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1982</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1983</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1984</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1985</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1986</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1987</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1988</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1989</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1990</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1991</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1992</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1993</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1994</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1995</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1996</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1997</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1998</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1999</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pBAddPerson">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_YearBorn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_YearDeath">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>115</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBMale">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>85</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rBFemale">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>85</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBBack">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_error_name">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>25</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_error_birth">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>55</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_error_death">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>115</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_error_turing">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_YearBorn</tabstop>
  <tabstop>rBMale</tabstop>
  <tabstop>rBFemale</tabstop>
  <tabstop>input_YearDeath</tabstop>
  <tabstop>cB_TuringYear</tabstop>
  <tabstop>input_Contribution</tabstop>
  <tabstop>pBAddPerson</tabstop>
  <tabstop>pBBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
